src/connector/src/parser/dynamodb/cdc_json_parser.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::error::ConnectorResult;
+use crate::only_parse_payload;
+use crate::parser::dynamodb::{build_dynamodb_json_accessor_builder, DynamodbChangeEvent};
+use crate::parser::unified::util::apply_row_operation_on_stream_chunk_writer;
+use crate::parser::{
+    AccessBuilderImpl, ByteStreamSourceParser, ParserFormat, SourceStreamChunkRowWriter,
+    SpecificParserConfig,
+};
+use crate::source::{SourceColumnDesc, SourceContext, SourceContextRef};
+
+#[derive(Debug)]
+pub struct DynamodbCdcJsonParser {
```

Comment:
Add sample data like json_parser?

Reply:
Oh, it's on `DynamodbChangeEvent`? Perhaps we can merge that file into this file, if it's only used for the CdcJsonParser

Reply:
updated.